jck/jck-semiauto-updater/jckupdater.sh

```diff
@@ -260,25 +341,27 @@ gitClone()
 copyFilestoGITRepo() {
 
 	cd $GIT_REPO
-	rm -rf JCK-compiler-$JCK_FOLDER_SUFFIX
-	rm -rf JCK-runtime-$JCK_FOLDER_SUFFIX
-	rm -rf JCK-devtools-$JCK_FOLDER_SUFFIX
-	rm -rf headers
-	rm -rf natives
-	echo "copy unpacked JCK files from $WORKSPACE/unpackjck to local GIT dir $GIT_REPO"
-
-	cd $WORKSPACE/unpackjck
-	for file in $WORKSPACE/unpackjck/*; do
-		echo $file
-		cp -rf $file $GIT_REPO
-	done
+	if [ $result -ne 0 ]; then
```

Comment:
Same here. if block should be added in https://github.com/adoptium/aqa-tests/blob/5e143d4d9370c9306fa5830d206cc80b1db58d22/jck/jck-semiauto-updater/jckupdater.sh#L440-L443

Reply:
Its required here as we need to copy files like .jtx, .kfl .html even if no JCK update available.

Reply:
re https://github.com/adoptium/aqa-tests/pull/4897/files#r1414368986, What I meant is that we perform the following only if result !=0.

https://github.com/adoptium/aqa-tests/blob/5e143d4d9370c9306fa5830d206cc80b1db58d22/jck/jck-semiauto-updater/jckupdater.sh#L440-L443

If statement should not be only added inside of `copyFilestoGITRepo()`. This is incorrect. The if statement should be added in https://github.com/adoptium/aqa-tests/blob/5e143d4d9370c9306fa5830d206cc80b1db58d22/jck/jck-semiauto-updater/jckupdater.sh#L440-L443